src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>MA-lesson1</title>
+</head>
+<body>
+    Hello, world!
+</body>
+</html>
```

Comment:
should be an empty line at the end of the file

Reply:
> should be an empty line at the end of the file

fixed